vars/kibanaPipeline.groovy

```diff
@@ -188,7 +188,7 @@ def withGcsArtifactUpload(workerName, closure) {
   def ARTIFACT_PATTERNS = [
     'target/junit/**/*',
     'target/kibana-*',
-    'target/kibana-coverage/**/*',
+    'target/kibana-coverage/jest/**/*',
```

Comment:
We should not upload `kibana-coverage/functional` content to artefacts since it is 30 GB of json files and we are not interested in them individually.